src/common/file_utils.cpp

```diff
@@ -4,15 +4,62 @@
 #include "common/string_utils.h"
 #include "glob/glob.hpp"
 
+#ifdef _WIN32
+#include <fileapi.h>
+#include <windows.h>
+#endif
+
 namespace kuzu {
 namespace common {
 
+FileInfo::~FileInfo() {
+#ifdef _WIN32
+    if (handle != nullptr) {
+        CloseHandle((HANDLE)handle);
+    }
+#else
+    if (fd != -1) {
+        close(fd);
+    }
+#endif
+}
+
+int64_t FileInfo::getFileSize() {
+#ifdef _WIN32
+    return GetFileSize((HANDLE)handle, nullptr);
```

Comment:
Should we use `GetFileSizeEx`? `GetFileSize` seems only return 4 bytes as the result, which is not large enough.